android/src/main/java/com/bitmovin/player/reactnative/converter/JsonConverter.kt

```diff
@@ -126,7 +126,8 @@ class JsonConverter {
             if (json.hasKey("customReceiverConfig")) {
                 customReceiverConfig = json.getMap("customReceiverConfig")
                     ?.toHashMap()
-                    ?.mapValues { entry -> entry.value as? String? } ?: emptyMap()
+                    ?.filterValues { entry -> entry is String  }
+                    ?.mapValues { entry -> entry.value as String } ?: emptyMap()
```

Comment:
Please be ruthless here, maybe there is better way to do this in Kotlin

Reply:
I don't think it's necessary to filter here as we can assume that the input on the ts side is correct. If we stop having this assumption the conversion will get super messy.
```suggestion
                    ?.mapValues { entry -> entry.value as String } ?: emptyMap()
```

Reply:
Sounds good!
Fixed by accepting your suggestion.